accounts/abi/unpack_test.go

```diff
@@ -290,6 +290,13 @@ var unpackTests = []unpackTest{
 			Int2 *big.Int
 		}{big.NewInt(1), big.NewInt(2)},
 	},
+	{
+		def: `[{"name":"int_one","type":"int256"}]`,
+		enc: "00000000000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000002",
```

Comment:
Could you please add the following tests:

  * One with `__`
  * One that has `int_one` and `intone`
  * One that yields nothing, e.g. `___`
  * One that ends with a `_`
  * One that has a potential conflict, like `int_one` and `intOne`.